app.json

```diff
@@ -9,12 +9,19 @@
     "repository": "https://github.com/symfony/symfony-demo",
     "logo": "https://symfony.com/images/v5/pictos/demoapp.svg?v=4",
     "success_url": "/",
+    "scripts": {
+        "postdeploy": "php bin/console doctrine:schema:create && php bin/console doctrine:fixtures:load -n"
```

Comment:
No. The name `postdeploy` for the script is, unfortunately, misleading; it is run [only after the very first deploy](https://devcenter.heroku.com/articles/app-json-schema#scripts) using the "Deploy to Heroku" button.